pom.xml

```diff
@@ -1,30 +1,43 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
     <modelVersion>4.0.0</modelVersion>
-    <groupId>aero.minova.rcp</groupId>
-    <artifactId>aero.minova.rcp.root</artifactId>
-    <version>12.7.4-SNAPSHOT</version>
-    <name>aero.minova.rcp</name>
-    <packaging>pom</packaging>
 
     <parent>
         <groupId>aero.minova</groupId>
-        <artifactId>spring.maven.root</artifactId>
-        <version>1.3.5</version>
+        <artifactId>maven.root</artifactId>
+        <version>12.4.3</version>
         <relativePath/>
     </parent>
 
+    <groupId>aero.minova.rcp</groupId>
+    <artifactId>aero.minova.rcp.root</artifactId>
+    <version>12.7.4-SNAPSHOT</version>
+    <packaging>pom</packaging>
+
+    <name>aero.minova.rcp</name>
+
     <properties>
-        <tycho.version>3.0.1</tycho.version>
-		<java.version>17</java.version>
-        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <java.version>17</java.version>
         <maven.javadoc.skip>true</maven.javadoc.skip>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
-        <sonar.coverage.jacoco.xmlReportPaths>
-            ${project.basedir}/report-aggregate/target/site/jacoco-aggregate/jacoco.xml
+        <sonar.coverage.jacoco.xmlReportPaths>${maven.multiModuleProjectDirectory}/report-aggregate/target/site/jacoco-aggregate/jacoco.xml
```

Comment:
`${maven.multiModuleProjectDirectory}` ist als "internal use only" markiert, aber Maven hat keinen guten Weg im Multimodule-Baum immer auf das Root-Module zu verweisen. Alles ist immer ein relativer Link in die nächsthöhere Hierarchie.